integration-tests/hibernate-orm-panache-kotlin/src/main/kotlin/io/quarkus/it/panache/kotlin/PersonRepository.kt

```diff
@@ -1,12 +1,12 @@
 package io.quarkus.it.panache.kotlin
 
 import io.quarkus.hibernate.orm.panache.kotlin.PanacheRepository
-import io.quarkus.hibernate.orm.panache.kotlin.runtime.JpaOperations
+import io.quarkus.hibernate.orm.panache.kotlin.runtime.KotlinJpaOperations.INSTANCE
 import javax.enterprise.context.ApplicationScoped
 
 @ApplicationScoped
 open class PersonRepository : PanacheRepository<Person> {
     override fun count(query: String, params: Map<String, Any>): Long {
-        return JpaOperations.count(Person::class.java, query, params)
+        return INSTANCE.count(Person::class.java, query, params)
```

Comment:
Tests should not be using `KotlinJpaOperations` but instead `super(query, params)`.

Reply:
Actually this method can probably come out.  I'd been directly implementing such methods so I had "real" bytecode to compare against.